src/main/java/frc/team2412/robot/commands/autonomous/AutonomousCommand.java

```diff
@@ -1,6 +1,95 @@
+        TrajectoryConfig config =
+                new TrajectoryConfig(
+                        Constants.AutoConstants.maxSpeedMetersPerSecond,
+                        Constants.AutoConstants.maxAccelerationMetersPerSecondSquared)
+                        // Add kinematics to ensure max speed is actually obeyed
+                        .setKinematics(Constants.DriveConstants.driveKinematics);
+
+        // An example trajectory to follow.  All units in meters.
+//        Trajectory exampleTrajectory =
+//                TrajectoryGenerator.generateTrajectory(
+//                        // Start at the origin facing the +X direction
+//                        new Pose2d(0, 0, new Rotation2d(0)),
+//                        // Pass through these two interior waypoints, making an 's' curve path
+//                        List.of(new Translation2d(1, 1), new Translation2d(2, -1)),
+//                        // End 3 meters straight ahead of where we started, facing forward
+//                        new Pose2d(3, 0, new Rotation2d(0)),
+//                        config);
+
+        Trajectory exampleTrajectory =
+                TrajectoryGenerator.generateTrajectory(
+                        // Start at the origin facing the +X direction
+                        new Pose2d(0, 0, new Rotation2d(0)),
+                        // Pass through these two interior waypoints, making an 's' curve path
+                        List.of(new Translation2d(-0.01, 0)),
+                        // End 3 meters straight ahead of where we started, facing forward
+                        new Pose2d(-0.3, 0, new Rotation2d(180)),
+                        config);
+
+
+
+//        ProfiledPIDController thetaController =
+//                new ProfiledPIDController(
+//                        Constants.AutoConstants.PThetaController, 0, 0, Constants.AutoConstants.kThetaControllerConstraints);
+//        thetaController.enableContinuousInput(-Math.PI, Math.PI);
+
+        ProfiledPIDController thetaController =
+                new ProfiledPIDController(
+                       0.000000005, 0, 0, Constants.AutoConstants.kThetaControllerConstraints);
+        thetaController.enableContinuousInput(-Math.PI, Math.PI);
+
+        SwerveControllerCommand swerveControllerCommand =
+                new SwerveControllerCommand(
+                        exampleTrajectory,
+                        drivebaseSubsystem::getPoseAsPose, // Functional interface to feed supplier
+                        Constants.DriveConstants.driveKinematics,
+
+                        // Position controllers
+                        new PIDController(Constants.AutoConstants.PXController, 0, 0),
+                        new PIDController(Constants.AutoConstants.PYController, 0, 0),
+                        thetaController,
+                        drivebaseSubsystem::updateModules,
+                        drivebaseSubsystem);
+
+
+        System.out.println(exampleTrajectory.getTotalTimeSeconds());
```

Comment:
It would be good to add some text to this, if it will be kept in the code base permanently.

Reply:
Noted, fixed extra spaced

Reply:
*spaces